tests/gtest/avifgainmaptest.cc

```diff
@@ -47,7 +46,6 @@ void CheckGainMapMetadataMatches(const avifGainMapMetadata& lhs,
 
 avifGainMapMetadata GetTestGainMapMetadata(bool base_rendition_is_hdr) {
   avifGainMapMetadata metadata = {};
-  metadata.backwardDirection = base_rendition_is_hdr;
```

Comment:
The `bool base_rendition_is_hdr` parameter is not used now. Should we remove it?

Reply:
Removed.